### DISCUSSION TITLE
💡 Feature Idea: protocol not required to add link

### DISCUSSION THREAD

**Discussion by fmerian** [dubinc/dub#158] [Ideas]
Currently, you must enter the protocol in the Destination URL to be able to add a new link: https://share.cleanshot.com/CWcKpQpC

**Comment by gabefletch** [comment]
Having the `https://` or `http://` protocol has always been required for Dub. I believe this was done for future-proofing the integration of other protocols like `mailto`, as discussed in #304. In this case it would make sense to need the protocol at the start of your target link to distinguish between these protocols.
